backend/internal/handlers/review/service.go

```diff
@@ -0,0 +1,135 @@
+package review
+
+import (
+	"context"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ReviewDocument struct {
+	ID        primitive.ObjectID `bson:"_id,omitempty" json:"_id,omitempty"`
+	Rating    Rating             `bson:"rating" json:"rating"`
+	Picture   string             `bson:"picture" json:"picture"`
+	Content   string             `bson:"content" json:"content"`
+	Reviewer  Reviewer           `bson:"reviewer" json:"reviewer"`
+	Timestamp time.Time          `bson:"timestamp" json:"timestamp"`
+	Comments  []interface{}      `bson:"comments,omitempty" json:"comments,omitempty"`
+	MenuItem  string             `bson:"menuItem" json:"menuItem"`
+}
+
+// Rating is a nested struct in ReviewDocument.
+type Rating struct {
+	Portion int `bson:"portion" json:"portion"`
+	Taste   int `bson:"taste" json:"taste"`
+	Value   int `bson:"value" json:"value"`
+	Overall int `bson:"overall" json:"overall"`
+	Return  int `bson:"return,omitempty" json:"return,omitempty"`
```

Comment:
I'm thinking we might want to alter "return" to be a boolean or constrain it to "0" or "1" rather than allow it to still be a [1,5] field. Do you also think omitempty is necessary here? I'm open to ideas